test/check-machines-lifecycle

```diff
@@ -83,8 +83,9 @@ class TestMachinesLifecycle(VirtualMachinesCase):
         self.login_and_go("/machines", user=user, superuser=superuser)
 
         if expect_empty_list:
-            b.wait_in_text("#virtual-machines-listing .pf-c-empty-state", "No VM is running")
-            return
+            with b.wait_timeout(20):
```

Comment:
Not sure if we really need that (CI machines can be quite busy), but as long as it works, fine for me.

Reply:
I noticed 1 times a test timing out without this at exactly this point. Let's leave it for now, and I might try to clean up more of these wait_timeout calls in a bulk cleanup.